infra/README.md

```diff
@@ -24,13 +41,15 @@
       - software selection: only SSH server and standard system utilities
 
 - Once the installation terminates and the VM has rebooted:
+  - if it is too slow, you can disable the graphical interface and work with command line: `sudo systemctl set-default multi-user` (becomes effective at restart)
```

Comment:
I'm not sure why the graphical interface would even be installed in the first place.
Even if it's the case, `systemctl disable gdm` should do the job, I'm not sure of the impact of changing the default boot target.

Reply:
"software selection: only SSH server and standard system utilities" => I probably didn't uncheck something during the installation. I will remove the line I added.